### ISSUE TITLE
Publish Event with raw data but not struct of CloudEvent

### ISSUE THREAD

**Issue by cvictory** [dapr/dapr#2736]
## In what area(s)?

 /area runtime
> /area operator
> /area placement
> /area docs
> /area test-and-release

## Describe the feature
We publish the pub/sub message with the struct ：
```
type CloudEvent struct {
	ID              string
	Data            []byte
	Topic           string
	Pubsub          string
	DataContentType string
	TraceID         string
}
```

But sometimes we expect we transfer the raw data, but not cloudEvent.

## Release Note

RELEASE NOTE:

**Comment by yaron2** [comment]
This is implemented as of 1.2.

**Comment by artursouza** [comment]
Issue is not resolved and also this is a duplicate of https://github.com/dapr/dapr/issues/2308
